modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -73,8 +73,9 @@ the *Status* column for any errors or failures.
 
 .. Click the installed *Cluster Logging* Operator.
 
-.. Under the *Overview* tab, click *Create Instance* . Paste the following YAML
-definition into the window that displays.
+.. Under the *Details* tab, in the *Provided APIs* section, in the
+*Cluster Logging* box, click *Create Instance* . Select *YAML View* and paste the following YAML definition into the window
```

Comment:
The "Select YAML View and paste the following YAML definition into the window..." should be on its own line as a separate step.

Reply:
Fixed!